README.md

```diff
@@ -67,7 +65,7 @@ create a new task, mark the task as completed  and delete them from local storag
 <!-- Live Demo -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-<a href="">Live-Demo</a>
+
```

Comment:
- [ ] [Optional] Well!👍 Considering that your project doesn't have a live version, it's advisable to remove the **Live Demo** section and the corresponding entry from the top list. This step will enhance the clarity and presentation of your project. Keep up the good work! 🚀